lib/model/migrations/20181206-01-add-projects.js

```diff
@@ -23,27 +23,31 @@ const up = (db) =>
   // and create the project actee species.
     .then(() => db.insert({ id: 'project', species: 'species' }).into('actees'))
 
-  // then create a default project. depending on whether forms already exist,
-  // the project's name will vary.
-    .then(() => Promise.all([
-      db.count('*').from('forms').where({ deletedAt: null }).limit(1)
-        .then(([{ count }]) => count),
-      db.insert({ id: uuid(), species: 'project' }).into('actees').returning('id')
-        .then(([ id ]) => id)
-    ])
-      .then(([ count, acteeId ]) => {
-        const name = (Number(count) > 0) ? 'Forms you made before projects existed' : 'Default Project';
-        return db.insert({ name, acteeId, createdAt: new Date() }).into('projects').returning('*');
+  // create a default project if forms already exist
+    .then(() => db.count('*').from('forms').where({ deletedAt: null }).limit(1)
```

Comment:
get rid of `deletedAt` in this query

Reply:
also what would happen if there were some forms but they were all deleted? would this all break? maybe